charts/invenio/templates/terminal-deployment.yaml

```diff
@@ -0,0 +1,169 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "invenio.fullname" . }}-terminal
+  labels:
+    {{- include "invenio.labels" . | nindent 4 }}
+    app.kubernetes.io/component: terminal
+spec:
+  {{- with .Values.terminal.deploymentSpec.minReadySeconds }}
+  minReadySeconds: {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.terminal.deploymentSpec.paused }}
+  paused: {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.terminal.deploymentSpec.progressDeadlineSeconds }}
+  progressDeadlineSeconds: {{- toYaml . | nindent 4 }}
+  {{- end }}
+  replicas: {{ .Values.terminal.replicas }}
+  {{- with .Values.terminal.deploymentSpec.revisionHistoryLimit }}
+  revisionHistoryLimit: {{- toYaml . | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: terminal
+  {{- with .Values.terminal.deploymentSpec.strategy }}
+  strategy: {{- toYaml . | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: terminal
+    spec:
+      {{- with .Values.terminal.nodeSelector }}
+      nodeSelector:
+        {{- tpl (toYaml .) $ | nindent 8 -}}
+      {{- end }}
+      {{- with .Values.terminal.tolerations }}
+      tolerations:
+        {{- tpl (toYaml .) $ | nindent 8 -}}
+      {{- end }}
+      containers:
+        - name: terminal
+          image: {{ default .Values.terminal.image (include "invenio.image" .) }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command: ['sleep']
+          args: ['infinity']
+          stdin: true
+          tty: true
+          resources: {{- toYaml .Values.terminal.resources | nindent 12 }}
+          securityContext:
+            {{- with .Values.terminal.securityContext }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "invenio.fullname" . }}-config
+            - secretRef:
+                name: {{ include "invenio.secretName" . }}
+            {{- with .Values.invenio.extraEnvFrom }}
+            {{- tpl (. | toYaml) $ | nindent 12 }}
+            {{- end }}
+            {{- with .Values.terminal.extraEnvFrom }}
+            {{- tpl (. | toYaml) $ | nindent 12 }}
+            {{- end }}
+          env:
+          - name: TZ
+            value: {{ required "Missing .Values.global.timezone" .Values.global.timezone }}
```

Comment:
I think it would be nicer to just have default values that sets this, rather than having it be "hard-coded". I don't feel strongly about it, WDYT?